k8s/demo/dbench/README.md

```diff
@@ -0,0 +1,128 @@
+This folder contains sample YAMLs for running bench marking tests
```

Comment:
Can we have folder structure as:
 `k8s/demo/perf/dbench`
or 
`k8s/perf/dbench`
or 
`perf/dbench`